java/src/main/java/ai/rapids/cudf/ParquetWriterOptions.java

```diff
@@ -56,6 +61,14 @@ public Builder withTimestampInt96(boolean int96) {
       return this;
     }
 
+    /**
+     * Flattened precision values for all the decimal columns
+     */
+    public Builder withPrecisionValues(int... precisionValues) {
```

Comment:
I really hate this API as a user facing API. I really would have preferred to see us expose a parquet like schema API. In the future we are going to need to pass down other things like is this list a byte array or a binary value. Is this list of structs really a map. Can we try and spend a little time to put the beginnings of that in place that we could then extend later on?

Reply:
Yes, its not the best. I thought about auto-generating this list when the parquet write method detects that it has decimals in the table. What do you think about that approach? 

I will think about it some more and think about future proofing this API

Reply:
I really want an API that exposes some kind of a ParquetSchema so we can give the parquet writer the extra metadata about the types that it is missing. I am fine if we don't do it now. It sounds like some of that work is going to happen at the C++ layer so for now we probably can just keep this API so long as we document it properly for what "flattened" really means.

Reply:
I have addressed the documentation on the method javadocs. 

Do you want me to create a follow-on issue for the Parquet-like API that you wanted?